docs/cloud/dbt-cloud-integration.md

```diff
@@ -0,0 +1,40 @@
+# Using the dbt Cloud integration
+
+## Step 1: Prerequisites
+ 
+### Generate a service token
+To generate a service token:
```

Comment:
Do they have instructions on generating a service token? If yes, delete this step and add a prerequisite linking to their official instructions.